spec.bs

```diff
@@ -292,6 +292,17 @@ Note: A later algorithm [=asserts=] that, for any [=contribution cache entry=]
     before the [=process contributions for a batching scope=] steps are
     performed given the entry's [=contribution cache entry/batching scope=].
 
+APIs exposing Private Aggregation {#apis-exposing-private-aggregation}
+----------------------------------------------------------------------
+
+<em>This section is non-normative.</em>
+
+This API is currently exposed in global scopes defined in two specifications:
+1. <a href="https://wicg.github.io/shared-storage/">Shared Storage</a>'s and
+1. <a href="https://wicg.github.io/turtledove/">Protected Audience</a>'s (via
```

Comment:
Optional nit: the possessives are a little jarring. Might read just as well without them since the noun before the colon is "specifications".

Reply:
Removed (and reworded "two specifications:" to "the specifications of two APIs:")